src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -1299,6 +1299,29 @@ theorem saddOverflow_eq {w : Nat} (x y : BitVec w) :
     simp
     omega
 
+theorem negOverflow_eq {w : Nat} (x : BitVec w) :
+    (negOverflow x) = ((0 < w) && (x = intMin w)) := by
+  simp only [negOverflow]
```

Comment:
```suggestion
theorem negOverflow_eq {w : Nat} (x : BitVec w) :
    negOverflow x = (decide (0 < w) && x == intMin w) := by
  simp only [negOverflow]
```
The parens on the lhs should be superfluous.
On the rhs, I prefer adding explicit `decide`s to emphasize where the `Bool`/`Prop` boundary happens. 
Also, I find `_ == _` nicer than `decide (_ = _)`, but that's largely up to taste.

I do wonder, could we simply use the rhs as the definition of `negOverflow`? I'd find it a nicer description, as it explicitly calls out that it's only for `intMin`, when the width is non-zero, rather than relying on `Int` arithmetic and an unnamed constant.

Reply:
thank you @alexkeizer for the review! I will indeed correct the bool/prop thing. 

The reason why I did not use `intMin` in the initial definition of `negOverflow` is to stick to the definition provided by the [SMTLIB standard](https://github.com/SMT-LIB/SMT-LIB-2/blob/2.7/Theories/FixedSizeBitVectors.smt2).